Assets/HoloToolkit/Input/Scripts/Utilities/AttachToController.cs

```diff
@@ -91,5 +91,11 @@ private void SetChildrenActive(bool isActive)
                 }
             }
         }
+        void Reset()
+        {
+            // We want the default value of Handedness of Controller finders to be Unknown so it doesn't attach to random object.
+            // But we also want the Editor to start with a useful default, so we set a Left handedness on inspector reset.
+            Handedness = UnityEngine.XR.WSA.Input.InteractionSourceHandedness.Left;
```

Comment:
breaks cross platform support.

Reply:
I haven't dealt with that yet. Do you mean to use an ifdef like this?
'''C#
#if UNITY_2017_2_OR_NEWER
using UnityEngine.XR.WSA.Input;
#else
using UnityEngine.VR.WSA.Input;
#endif
'''

Reply:
yes, But I believe there's already a platform agnostic handedness enum you can use instead.